dotcom-rendering/src/web/components/SignInGate/gates/main-variant.tsx

```diff
@@ -15,19 +15,66 @@ const SignInGateMain = React.lazy(() => {
 	});
 });
 
+const SignInGateMainCheckoutComplete = React.lazy(() => {
+	const { start, end } = initPerf('SignInGateMainCheckoutComplete');
+	start();
+	return import(
+		/* webpackChunkName: "SignInGateMain" */ '../gateDesigns/SignInGateMainCheckoutComplete'
```

Comment:
Are we intentionally using the same webpack chunk as the regular `SignInGateMain`? Would be be better to have these in separate chunks?

Reply:
Oh that's a good spot, thank you. These should be different